pkg/network/cluster_config.go

```diff
@@ -206,18 +211,43 @@ func validateOVNKubernetesCIDROverlap(clusterCofnig *configv1.Network, operConfi
 	return nil
 }
 
+const metricLiveMigrationBlockedLabelKey = "reason"
+
+// openshift_network_operator_live_migration_blocked metric 'reason' label key name values
+const (
+	unsupportedCNI                     = "UnsupportedCNI"
+	unsupportedHCPCluster              = "UnsupportedHyperShiftCluster"
+	unsupportedSDNNetworkIsolationMode = "UnsupportedSDNNetworkIsolationMode"
+	unsupportedMACVlanInterface        = "UnsupportedMACVLANInterface"
```

Comment:
We block migration when an egress router is configured. So, we should name the reason more relevant.